test/battle/move_effect/ivy_cudgel.c

```diff
@@ -28,3 +28,24 @@ SINGLE_BATTLE_TEST("Ivy Cudgel changes the move type depending on the mask the u
         MESSAGE("It's super effective!");
     }
 }
+
+SINGLE_BATTLE_TEST("Ivy Cudgel does not change the move type if used on Pokémon other than Ogerpon")
```

Comment:
```suggestion
SINGLE_BATTLE_TEST("Ivy Cudgel does not change the move type if used by Pokémon other than Ogerpon")
```
"on" could be interpreted as targets